packages/edit-site/src/components/page-patterns/grid-item.js

```diff
@@ -154,7 +154,9 @@ function GridItem( { categoryId, item, ...props } ) {
 						: undefined
 				}
 			>
-				{ isEmpty && __( 'Empty pattern' ) }
+				{ isEmpty && isTemplatePart
+					? __( 'Empty template part' )
+					: __( 'Empty pattern' ) }
```

Comment:
```suggestion
				{ isEmpty && isTemplatePart && __( 'Empty template part' ) }
				{ isEmpty && ! isTemplatePart && __( 'Empty pattern' ) }
```

I think the conditional statement would be like this suggestion, for example, because this text should only be rendered if `isEmpty` is true. Otherwise, unintended text will be rendered.

![empty-pattern](https://github.com/WordPress/gutenberg/assets/54422211/74d47818-b80e-4405-8e73-445e0134d757)

Reply:
🤦 Yep, thanks. I rushed the tweak before finishing up for the day yesterday.